.gitsubprojects

```diff
@@ -1,5 +1,5 @@
 # -*- mode: cmake -*-
-git_subproject(ZeroEQ https://github.com/HBPVIS/ZeroEQ.git 1949f24)
+git_subproject(ZeroEQ https://github.com/HBPVIS/ZeroEQ.git ff94fa8)
```

Comment:
That's annoying... Then you need to do this too: https://github.com/BlueBrain/Tide/pull/167/files#diff-54d15a4c272c45dd76aad2b4962b05d2R88